lib/message_express/bus/rd_kafka.rb

```diff
@@ -0,0 +1,37 @@
+require 'message_express/bus/abstract_bus'
+require 'message_express/message'
+
+module MessageExpress
+  module Bus
+    class RdKafka < AbstractBus
+      attr_reader :kafka
+
+      def initialize(kafka_client, topic: nil)
+        @kafka = kafka_client
+        @topic = topic || 'messages'
+      end
+
+      def publish(message)
+        producer_params = {
+          topic: @topic,
+          payload: message,
+          key: "#{@topic}-#{rand(10**10)}"
+        }
+
+        producer = @kafka.producer
+        producer.produce(producer_params).wait
+      end
+
+      def subscribe(consumer_group_id:, topic: @topic)
+        consumer = @kafka.consumer(group_id: consumer_group_id)
+        consumer.subscribe(topic)
+
+        trap('TERM') { consumer.stop }
+
+        consumer.each do |kafka_message|
+          Rails.logger.info "Message: #{kafka_message}"
```

Comment:
You don't need to mark this message as processed?

Reply:
There is a way to know if a message have been listened already?

I also think we should do something with the message. 
```ruby
message_payload = JSON.parse(kafka_message.payload)
message = Message.new(kafka_message.key, message_payload)
yield(message)
```

Reply:
this subscriber is for the test proposes only, other applications will process them.